cmd/mirror-url.go

```diff
@@ -47,31 +47,39 @@ func checkMirrorSyntax(ctx *cli.Context, encKeyDB map[string][]prefixSSEPair) {
 		errorIf(errInvalidArgument().Trace(URLs...), "`--force` is deprecated please use `--overwrite` instead for the same functionality.")
 	}
 
+	if len(tgtURL) == 0 && tgtURL == "" {
```

Comment:
aren't these both same?

Reply:
> aren't these both same?

yes, this code was just moved but I can fix it

Reply:
Actually I am removing that code, it is not necessary, we will never have tgtURL to be empty since we already check the number of arguments in the beginning of checkMirrorSyntax()

Reply:
> Actually I am removing that code, it is not necessary, we will never have tgtURL to be empty since we already check the number of arguments in the beginning of checkMirrorSyntax()

Fixed